index.html

```diff
@@ -3396,11 +3431,13 @@ <h2>
           identifier</a> for the <a>payment method</a> that the user selected
           to accept the payment.
           </li>
+          <li>Let <var>handler</var> be the <a>payment handler</a> that the
+          user is interacting with via <var>request</var>.
+          </li>
```

Comment:
Could we say "Let <var>handler</var> be the <a>payment handler</a> selected by the user." ? 
For some envisioned cases of frictionless payments, there may not be interaction other than selection by the 
user.